include/openssl/safestack.h

```diff
@@ -32,104 +32,104 @@ extern "C" {
     typedef t3 * (*sk_##t1##_copyfunc)(const t3 *a); \
     static ossl_unused ossl_inline int sk_##t1##_num(const STACK_OF(t1) *sk) \
     { \
-        return OPENSSL_sk_num((const OPENSSL_STACK *)sk); \
+        return OPENSSL_sk_num((const OPENSSL_STACK *)(const void *)sk); \
```

Comment:
Why is this double cast necessary? Same question for all the similar occurrences below.

Reply:
These are needed for -fstrict-aliasing

Reply:
Ok